pkg/core/ca/provided/rest/client.go

```diff
@@ -5,6 +5,7 @@ import (
 	"encoding/json"
 	"fmt"
 
+	kumactl_config "github.com/Kong/kuma/pkg/config/app/kumactl/v1alpha1"
```

Comment:
Let's avoid dependency ` pkg/core => pkg/config/app/kumactl`.

Probably, we should move this client into `app/kumactl`